y/y.go

```diff
@@ -515,3 +515,23 @@ func NewKV(alloc *z.Allocator) *pb.KV {
 	b := alloc.AllocateAligned(kvsz)
 	return (*pb.KV)(unsafe.Pointer(&b[0]))
 }
+
+// IBytesToString converts size in bytes to human readable format.
+// The code is copied from humanize library and changed to provide
+// value upto 1 decimal place when it exceeds 1 GiB.
+func IBytesToString(size uint64) string {
```

Comment:
Maybe rename it to `IBytes`. Also, add an example in the function description
// IBytes(xxxx) -> xx.xx GiB

Reply:
IBytes doesn't sounds descriptive enough. Added the usage example.